cli/src/build/onboarding/android/ui/app.tsx

```
@@ -966,8 +1028,22 @@ const AndroidOnboardingApp: FC<AppProps> = ({ appId, initialProgress, androidDir
           // yet — the wizard now offers that step only AFTER a successful first
           // build, so users never end up with orphan secrets in a repo whose
           // build was never proven to work.
-          const entries = createCiSecretEntries(credentials)
+          //
+          // Pass the API key so CAPGO_TOKEN gets included — the generated
+          // GitHub Actions workflow references ${{ secrets.CAPGO_TOKEN }} for
+          // --apikey, and users who pick "secrets only" still benefit from
+          // having it ready in their repo for a workflow they'll write later.
+          let capgoKey: string | undefined = apikey
+          if (!capgoKey) {
+            try { capgoKey = findSavedKey(true) }
```

ESLint

[error] This line has 2 statements. Maximum allowed is 1.

(style/max-statements-per-line)